master/render.html

```diff
@@ -638,7 +638,7 @@ <h3 id="PaintingRasterImages">Painting raster images</h3>
 requirements are discussed under
 <a href="conform.html">conformance requirements</a>.</p>
 <p class="ready-for-wider-review">
-As in HTML [<a href="refs.html#ref-html51">HTML</a>, 10.4.2],
+As in HTML [<a href="refs.html#ref-html">HTML</a>, 10.4.2],
```

Comment:
With a belated review comment: that section number is wrong. Haven't tracked down the correct one.

If you have a chance, it could be fixed as a separate commit.